addons/stock_dropshipping/tests/test_dropship.py

```diff
@@ -350,3 +350,30 @@ def test_add_dropship_product_to_subcontracted_service_po(self):
             {'product_id': subcontracted_service.id, 'product_uom_qty': 1.0, 'qty_delivered': 0.0},
             {'product_id': self.dropship_product.id, 'product_uom_qty': 0.0, 'qty_delivered': 1.0},
         ])
+
+    def test_dropship_lot_product_appears_in_stock_lot_report(self):
+        dropship_product = self.lot_dropship_product
+        sale_order = self.env['sale.order'].create({
+            'partner_id': self.customer.id,
+            'order_line': [Command.create({
+                'product_id': dropship_product.id,
+                'product_uom_qty': 2,
+            })],
+        })
+        sale_order.action_confirm()
+        purchase_order = self.env['purchase.order'].search([('partner_id', '=', self.supplier.id)])
+        purchase_order.button_confirm()
+        dropship_picking = purchase_order.picking_ids
+        dropship_picking.move_line_ids.lot_name = 'dropship product lot'
+        dropship_picking.move_ids.picked = True
+        dropship_picking.button_validate()
+        self.env.flush_all()
```

Comment:
Could be better to limit the flush here ? 
I mean, only with the models we need ? Like SML and maybe SO ?